Commands.adoc

```diff
@@ -0,0 +1,239 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you `vagrant up` and have a running VM, the vagrant-service-manager
+ enables you to check and tweak your container development environment.
+
+'''
+toc::[]
+'''
+
+You can use the vagrant-service-manager to set up your environment variables and
+ get the TLS certificates to secure the Docker communication channel; identify
+ the routable ip network and version of your VM; and check the status and
+ manage the lifecycle of the configured services in the running VM.
+
+To run the vagrant-service-manager plugin use: +
+-------------------------------------------
+vagrant service-manager [command] [options]
+-------------------------------------------
+The following commands can be used: +
+`env`: Displays the connection information for services in the VM +
+`VM`: Displays the VM related information like version, release, IP etc +
+`status`: Lists services and their running state +
+`start`: Starts the given service in the VM +
+`stop`: Stops the given service in the VM +
+`restart`: Restarts the given service in the VM +
+
+The following options can be used: +
+`--script-readable`: Displays information in a key=value format +
```

Comment:
maybe we can try a labeled list here as well!? Let's give it a go and see how it looks like.